azure_monitor/src/azure_monitor/export/trace/__init__.py

```diff
@@ -56,3 +55,105 @@ def _span_to_envelope(self, span: Span) -> protocol.Envelope:
         envelope = convert_span_to_envelope(span)
         envelope.ikey = self.options.instrumentation_key
         return envelope
+
+
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-branches
+def convert_span_to_envelope(span: Span) -> protocol.Envelope:
```

Comment:
is there any change in this code apart of the location change?

Reply:
Nope.